packages/react/src/components/SignIn/fragments/basic-auth.scss

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.asgardeo-basic-auth-skeleton {
+  display: flex;
+  flex-flow: column nowrap;
+  padding: 32px;
+  row-gap: 8px;
+
+  .skeleton-title {
+    margin: 0 auto;
+  }
+
+  .skeleton-text-field-label {
+    margin-top: 12px;
+  }
+
+  .skeleton-submit-button {
+    margin: 20px auto 0;
+    border-radius: 20px;
+  }
+}
+
+.asgardeo-basic-auth-paper {
+  opacity: 0.5;
+  animation: fade-in 0.9s ease-in-out forwards;
+
+  .asgardeo-basic-auth-alert {
+    margin-top: 12px;
+  }
+}
+
+@keyframes fade-in {
+  to {
+    opacity: 1;
+  }
+}
```

Comment:
Better not to add animations IMO as this could clash with the application's aesthetic. That should be on the hands of the app developer